[IDEA] Mark used parameters as disabled in parameter info popup

cketti opened the pull request:
When using named arguments the IDE might be unable to map the current argument to a parameter. In that case this change marks already used parameters as disabled. That way developers can easily see which parameters have not been used yet.

![idea_parameter_info_unused_parameters_variant_1](https://user-images.githubusercontent.com/218061/92270909-22632380-eee7-11ea-84ec-6cd4b99a9515.png)


Fixes [KTIJ-128](https://youtrack.jetbrains.com/issue/KTIJ-128)

vladimirdolzhenko commented:
I would like to recommend to add one more test to cover this specific case

Like 
```
fun foo(a: String = "a", b: String = "b", c: String = "z") {}

val f = foo(b = "z", c = "z", <caret>)
```

cketti commented:
Isn't that the same as [NamedParameter3.kt](https://github.com/JetBrains/kotlin/blob/54a0efb340dffcebaf0628afa394eb3847479d28/idea/testData/parameterInfo/functionCall/NamedParameter3.kt)?

vladimirdolzhenko commented:
similar, but NamedParameter3.kt is a bit unclear as there are only 2 parameters - maybe it's worth to make for 4 params and use 2 of them ?
As well in test data of NamedParameter3.kt it's not clear those parameters are disabled - maybe it's the matter of rendering disabled parameters in test data and would be nice to adjust them as well.

cketti commented:
I added another test with 4 named parameters and 2 of them used, like you suggested.

I don't know if this is what you meant, but I changed `MockParameterInfoUIContext` to emit `<disabled>` tags. The expected output for `NamedParameter3.kt` is now:

```
Text: (<disabled>[y: Boolean],</disabled><highlight> </highlight>[x: Int]), Disabled: false, Strikeout: false, Green: true
```

vladimirdolzhenko commented:
You get it right, thank you.

vladimirdolzhenko commented:
LGTM
